lambdas/lambdas.mk

```
@@ -28,9 +28,13 @@ include $(abspath $(dir $(lastword $(MAKEFILE_LIST))))/../common.mk
 # Set literals will compile in a non-deterministic order unless PYTHONHASHSEED
 # is set. For a full explanation see http://benno.id.au/blog/2013/01/15/python-determinism
 #
+# `compileall` ignores symlinks when traversing directories, so we must
+# explicitly list them as arguments to ensure all files are in vendor/ are
+# deterministically compiled.
+#
 .PHONY: compile
 compile: check_python
-	PYTHONHASHSEED=0 python -m compileall -f -q --invalidation-mode checked-hash vendor vendor/azul
+	PYTHONHASHSEED=0 python -m compileall -f -q --invalidation-mode checked-hash vendor $(shell find vendor -maxdepth 1 -type l)
```

Let's exclude symbolic links that do not point to directories.

```suggestion
	PYTHONHASHSEED=0 python -m compileall -f -q --invalidation-mode checked-hash vendor $(shell find -f $$(find vendor -maxdepth 1 -type l) -maxdepth 0 -follow -type d)
```

---

But please confirm by experiment that compileall does not ignore symlinks that 1) point at files and 2) are discovered during traversal.

---

When I try to run the `compile` target with this patch, I get `find: unknown predicate `-f'`. Additionally, the `man` page says that `-follow` is deprecated. `find` version 4.9.0. It seems to work as intended if I omit the `-f` parameter.